modules/backend/widgets/reportcontainer/assets/js/reportcontainer.js

```diff
@@ -41,7 +41,7 @@
         $(window).resize($.proxy(this.updateWidth, this))
         this.updateWidth()
 
-        if (!Modernizr.touchevents) {
+        if (!Modernizr.touch) {
```

Comment:
@ayumihamsaki touch event was removed from the `Modernizr`, could you review your PR? Maybe there are some old leftovers :)

Reply:
Have updated a new commit and also spotted a couple of typo's in the code